sormas-base/setup/generate-cert.sh

```diff
@@ -0,0 +1,91 @@
+
+#*******************************************************************************
+# SORMAS® - Surveillance Outbreak Response Management & Analysis System
+# Copyright © 2016-2018 Helmholtz-Zentrum f�r Infektionsforschung GmbH (HZI)
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#*******************************************************************************
+
+#!/bin/bash
+
+echo "# SORMAS TO SORMAS NEW CERTIFICATE GENERATION"
+echo "# This script generates a new self signed certificate, to be used for SORMAS2SORMAS and SurvNet communication"
+echo "# If anything goes wrong, please consult the certificate creation guide or get in touch with the developers."
+
+if [[ $(expr substr "$(uname -a)" 1 5) = "Linux" ]]; then
+	LINUX=true
+else
+	LINUX=false
+fi
+
+# DIRECTORIES
+if [[ ${LINUX} = true ]]; then
+	ROOT_PREFIX=
+else
+	ROOT_PREFIX=/c
+fi
+SORMAS2SORMAS_DIR=${ROOT_PREFIX}/opt/sormas2sormas
```

Comment:
I would prefer to only set the variable when it has not been set before.
We could also ask for it if the directory doesn't exist.
Also shouldn't it be `/opt/sormas/sormas2sormas` ?

Reply:
- How can this variable be set before? Should it be an environment variable?
- Do you mean if the root directory doesn't exist, ask for a path to the sorms2sormas directory?

Reply:
Maybe something like that?
```
if [[ -z "${SORMAS2SORMAS_DIR}" ]]; then
  DEFAULT_SORMAS2SORMAS_DIR="${ROOT_PREFIX}/opt/sormas2sormas"
  if [[ -d "${DEFAULT_SORMAS2SORMAS_DIR}" ]]; then
    SORMAS2SORMAS_DIR="${DEFAULT_SORMAS2SORMAS_DIR}"
  else
    	while [[ ! -d "${SORMAS2SORMAS_DIR}" ]]; do
		read -r -p "Please specify a valid sormas2sormas directory: " SORMAS2SORMAS_DIR
	done
  fi
fi
```